packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -334,17 +333,33 @@ export const hideScrollbar = (): SerializedStyles => css`
 `;
 
 /**
- * Visually communicates to the user that an input is hovered, focused,
- * or active.
+ * Visually communicates to the user that an element hovered, focused, or
+ * active; and optionally adds disabled, invalid or warning styles.
  */
 export const inputOutline = (
   args:
     | Theme
     | {
+        /**
+         * The Theme object.
+         */
         theme: Theme;
+        /**
+         * Adds disabled styles.
+         */
         disabled?: boolean;
+        /**
+         * Adds invalid styles.
+         */
         invalid?: boolean;
+        /**
+         * Adds warning styles.
+         */
         hasWarning?: boolean;
+        /**
+         * @deprecated
+         * Has no effect, kept for legacy reasons.
```

Comment:
As discussed. I thought that marking this as deprecated could ensure that people don't use this by mistake, what do you think?

Reply:
Makes sense. We can un-deprecate it in the future if we need it after all.